ddtrace/bootstrap/sitecustomize.py

```diff
@@ -43,6 +43,22 @@ def update_patched_modules():
         EXTRA_PATCHED_MODULES.update({module: should_patch.lower() == 'true'})
 
 
+def add_global_tags(tracer):
+    tags = {}
+    for tag in os.environ.get('DD_TRACE_TAGS', '').split(','):
+        if len(tag.split(':')) != 2:
+            log.debug("skipping malformed tracer tag")
+            continue
+
+        tag_name, tag_value = tag.split(':')
+        if tag_value.lower() in ['true', 'false']:
```

Comment:
All tags end up getting added as strings anyways, so we can probably skip these conversions.

https://github.com/DataDog/dd-trace-py/blob/4e5acbd8ca1d4087291d9d9c1709b61651bba230/ddtrace/span.py#L127-L135